### ISSUE TITLE
SWbemObjectSet: Invalid class runtime error at install (fail)

### ISSUE THREAD

**Issue by idheitmann** [AntonelloDN/Morpho#17]
runtime error at 9:841

**Comment by AntonelloDN** [comment]
Hi @idheitmann, can you elaborate more? 
Do you have an old version of Morpho which was installed without the installer on your PC?

**Comment by idheitmann** [comment]
Yes, I had old Morpho still. I deleted the Morpho folder from AppData as well as all the user objects. This is still what i get:

![image](https://github.com/AntonelloDN/Morpho/assets/14552260/078f8309-ae2d-4327-8ad5-5bb2e93374aa)

**Comment by idheitmann** [comment]
i have tried compatibility mode as well as tried running as administrator and checked the filesystem permissions

**Comment by DeernsDHG** [comment]
Hello idheitman, I am having the same trouble on laptop of user. I tried to start it as administrator then it says:
Rhino and Grashopper not installed.
Also tried older versions but not succesfull.
Are you still having this issue

**Comment by lebemann** [comment]
Hello everyone, I got the same error. Does anyone know how to solve it ?

**Comment by kathlau** [comment]
Try to re-register MOF Files

1. Open the Command Prompt as an administrator:
Press Win + S, type cmd, then right-click on Command Prompt and select Run as Administrator.
Execute the following command to navigate to the WMI folder:

2. Execute the following command to navigate to the WMI folder:
cd C:\Windows\System32\wbem

3. Run this command to re-register all MOF (Managed Object Format) files:
for /f %s in ('dir /b *.mof *.mfl') do mofcomp %s
